TODO

```diff
@@ -3,9 +3,9 @@
    - maybe we can get rid of the heartbeat thread and just have the
      reader thread do zero length writes every so often?
 - Call `setsid()` (`man 2 setsid`) in the fork child before exec. This creates
```

Comment:
This file is about to get clobbered in https://github.com/shell-pool/shpool/pull/34, so just leave it as is.